integration_tests/dbt_project/macros/replace_empty_strings_with_nulls.sql

```diff
@@ -0,0 +1,18 @@
+{% macro replace_empty_strings_with_nulls(table_name) %}
```

Comment:
This is a test macro that bypasses an issue with BQ seeds in fusion - where nulls are loaded as empty strings (it messed up the tests as some of them use not_null tests - so I just created this macro to change empty strings to nulls)